delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/ElasticSearchScopeInitialization.scala

```diff
@@ -0,0 +1,74 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch
+
+import ch.epfl.bluebrain.nexus.delta.kernel.syntax._
+import ch.epfl.bluebrain.nexus.delta.kernel.utils.ClasspathResourceUtils
+import ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.model.ElasticSearchViewRejection.ViewAlreadyExists
+import ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.model.ElasticSearchViewValue.IndexingElasticSearchViewValue
+import ch.epfl.bluebrain.nexus.delta.rdf.Vocabulary.nxv
+import ch.epfl.bluebrain.nexus.delta.sdk.error.ServiceError.ScopeInitializationFailed
+import ch.epfl.bluebrain.nexus.delta.sdk.model.IdSegment
+import ch.epfl.bluebrain.nexus.delta.sdk.model.IdSegment.IriSegment
+import ch.epfl.bluebrain.nexus.delta.sdk.model.identities.{Caller, Identity, ServiceAccount}
+import ch.epfl.bluebrain.nexus.delta.sdk.model.organizations.Organization
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.Project
+import ch.epfl.bluebrain.nexus.delta.sdk.{MigrationState, ScopeInitialization}
+import com.typesafe.scalalogging.Logger
+import io.circe.Json
+import monix.bio.{IO, UIO}
+
+/**
+  * The default creation of the default ElasticSearchView as part of the project initialization. It performs a noop if
+  * executed during a migration.
+  *
+  * @param views          the ElasticSearchViews module
+  * @param serviceAccount the subject that will be recorded when performing the initialization
+  */
+class ElasticSearchScopeInitialization(views: ElasticSearchViews, serviceAccount: ServiceAccount)
+    extends ScopeInitialization {
+
+  private val logger: Logger          = Logger[ElasticSearchScopeInitialization]
+  implicit private val caller: Caller = serviceAccount.caller
+
+  private def loadDefault(resourcePath: String): IO[ScopeInitializationFailed, Json] =
+    ClasspathResourceUtils
+      .ioJsonContentOf(resourcePath)(getClass.getClassLoader)
+      .mapError(e => ScopeInitializationFailed(e.toString))
+      .memoize
+
+  private val defaultViewId: IdSegment                             = IriSegment(nxv + "defaultElasticSearchIndex")
```

Comment:
put this value in the package object in `ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.model` since I'm gonna need it